KosherCocoa/Library/Core/Calendar/Chagim/KCJewishCalendar.m

```diff
@@ -662,7 +672,12 @@ - (BOOL)isHebrewLeapYear:(NSInteger)year
 //Determine if kislev is short this year
 - (BOOL)isKislevShort
 {
-    return [self lengthOfHebrewYear:[self currentHebrewYear]] % 10 == 3;
+    return [self lengthOfHebrewYear:[self currentHebrewYear]] == KCHebrewYearTypeChaser;
+}
+
+- (BOOL)isCheshvanLong
+{
+    return [self lengthOfHebrewYear:[self currentHebrewYear]] == KCHebrewYearTypeShalaim;
 }
```

Comment:
Added a method here because of simplicity. I don't know why the year types were not used.

Reply:
In future code that I have not uploaded yet